src/preprocessing/subtract_black.cpp

```diff
@@ -39,26 +39,26 @@ int LibRaw::subtract_black_internal()
       int dmax = 0;
       if (C.cblack[4] && C.cblack[5])
       {
-        for (i = 0; i < size * 4; i++)
-        {
-          int val = imgdata.image[0][i];
-          val -= C.cblack[6 + i / 4 / S.iwidth % C.cblack[4] * C.cblack[5] +
-                          i / 4 % S.iwidth % C.cblack[5]];
-          val -= cblk[i & 3];
-          imgdata.image[0][i] = CLIP(val);
-          if (dmax < val)
-            dmax = val;
+        for (unsigned i = 0; i < size; i++) {
```

Comment:
Should probably match the existing libraw style of parens on newlines

Reply:
Good catch!